lacasera/laravel-emojis
# #3 Add Animal Emogis

### Please check if the PR fulfills these requirements


### What does this PR do
Adds Animals emogis 

### Steps to manually test PR
1. run `getEmoji('joy_cat')` on an instance of the `Emoticons` class
2. notice 😹 is returned

### Related Issue
#3 

### Other information
